src/utils/make-styles.js

```diff
@@ -0,0 +1,7 @@
+import { css } from 'goober'
+
+
+export const makeStyles = (classObj) => Object.entries(classObj).reduce((acc, [key, val]) => {
```

Comment:
this is cool but where is this used?

Reply:
for backup in case we need to use some in-line styling, also for use in projects that uses lumen-labs and needs in-line styling

Reply:
sounds good. then iteratively we may be able to work in dynamic aspects like in `styled-components` or MUI's equiv

Reply:
yea 😃